src/main/resources/META-INF/services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+com.facebook.presto.maxcompute.MaxComputePlugin
```

Comment:
Can you use the presto-maven-plugin to add this so you don't need to add it by hand?

Reply:
OK, I am testing the integration of my Runner into `AbstractTestIntegrationSmokeTest`. I found that `AbstractTestIntegrationSmokeTest` requires a TPCH dataset in my database emulator, so I generated one with TPCH datagen. However, I found that the generated dataset does not match the test expectations. For example, in the `testAggregateSingleColumn` test, there is a java.lang.AssertionError:

```
For query:
 SELECT SUM(orderkey) FROM orders
not equal
Actual rows (1 of 1 extra rows shown, 1 rows in total):
    [4499987250000]
Expected rows (1 of 1 missing rows shown, 1 rows in total):
    [449872500]
```

Can you tell me how the TPCH dataset required by this test is generated? Does it use a sample ratio of 0.001?

Reply:
[We use the tiny scale factor, which is 0.01.](https://github.com/prestodb/presto/blob/8f26ac6548ff738940ffa4e9b34fed209878a22e/presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java#L96-L97)